lessons/Data-Science/histograms/langs/en-us/pages/data-cycle-histograms-animals.adoc

```diff
@@ -12,20 +12,21 @@ table.template td:first-of-type { background: #ccc; }
 </style>
 ++++
 
-Use the Data Cycle to explore the distribution of one or more quantitative columns using *histograms*, and write down your findings.
+Use the Data Cycle to explore the distribution of one or more quantitative columns in @starter-file{animals} using *histograms*, and write down your findings.
 
 @data-cycle{ ""
   #:question "What is the shape of the age column of the Animals dataset?"
   #:show-question? #t
-  #:rows ""
+  #:question-type "statistical"
+  #:rows "All the rows"
   #:show-rows? #f
-  #:cols ""
+  #:cols "age"
   #:show-cols? #f
   #:filter-fn ""
   #:show-filter? #f
   #:build-fn ""
   #:show-build? #f
-  #:expression ""
+  #:expression  (if (string=? *proglang* "pyret") '(histogram animals-table "name" "age" 3) "histogram")
```

Comment:
Confirming that this is because you want a version that works for codap?

Reply:
yeah, but I should give a better CODAP answer. Fixing.

Reply:
Is there a better way to do this? Just built and the improved CODAP explanation shows up in code font, which is too big for the line given the amount of text?

(I reduced the amount of text as a maybe temporary solution?)

Reply:
@retabak Done! See f18febeac